llvm/docs/AMDGPUUsage.rst

```diff
@@ -1755,6 +1755,56 @@ As part of the AMDGPU MC layer, AMDGPU provides the following target specific
 
      =================== ================= ========================================================
 
+Function Resource Usage
+-----------------------
+
+The function resource information (e.g., number of VGPRs) required depends on
+all of its callees' function resources. The expression to denote these
+resources should, therefore, be described as the propagative of its callees'
+equivalent expressions. Said expressions are generated and emitted (as symbols)
+by the compiler when compiling to either assembly or object format and should
+not be overwritten or redefined.
+
+The following describes all emitted function resource usage information:
```

Comment:
Here too I think "information" could be replaced with "symbols". Then there are only two terms to keep track of:

* "Resource usage": The concept
* "Resource usage symbols": The interface we expose to access the usage